### DISCUSSION TITLE
Adding tailwind to only specific part of the app.

### DISCUSSION THREAD

**Discussion by BornaMarin** [tailwindlabs/tailwindcss#2443] [Help] [answered]
Hello, i'm new to tailwind. What i'm interested in is adding tailwind to only specific part of my app. Project i'm working on is 2 years old and full of vanilla css and tons of libraries.  Is there any  way to enable tailwind only on specific directory/file. I'm using nuxt js(vue.js).

**Comment by paulpopus** [comment, marked as answer]
Tailwind just provides you with a compiled CSS file on the output, you can then include that anywhere you want, it doesn't have to be included everywhere, just as you would with any CSS file or other assets.

Your automatic building with Nuxt (I assume webpack) will need to be modified so Tailwind will run in its own build step so you get a different CSS output.

Depends how complicated you want to make it.

**Reply by devgauharnawab** [reply]
I'd really appreciate it if you could explain how I can apply Bootstrap and Tailwind CSS on a particular component. It would be a great help and save me a lot of time. Thank you!
